src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/OptionsPostAndReply.java

```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)